gandalf/src/main/java/io/github/btkelly/gandalf/Gandalf.java

```diff
@@ -106,6 +131,7 @@ public void onSuccess(Bootstrap bootstrap) {
 
                     @Override
                     public void onError(IOException e) {
+                        LoggerUtil.logE("Error fetching bootstrap: " + e.getMessage());
                         //In any error case we should let the user in as to not block based on a bug
```

Comment:
And didn't know if we wanted something like Amplify has where the logger is injected all over instead of static calls